### PR TITLE
Config panel 6

### LINKED ISSUE #6: UI configuration panel
At present the `Export` button generates images from the fitting results and will save a local copy of the configuration file if the `Save config` checkbox is checked. It would be better to have a separate panel for controls related to saving and loading configuration settings.

### LINKED ISSUE #13: Kernel size does not get saved in exported config
Kernel size is the only UI parameter that doesn't get saved into the config file when exporting.

### PR DESCRIPTION
**Pull request by drmatthews**
This adds a config panel to the UI with `Load Config` and `Save Config` buttons and removes the `Save Config` checkbox from the `Export` panel.

This also adds `binRadius` and `FitType` fields to the config file so that the UI can be fully restored when loading a configuration.

Closes #6 and closes #13

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -94,9 +94,10 @@
 		<!-- NB: Deploy releases to the SciJava Maven repository. -->
 		<releaseProfiles>deploy-to-scijava</releaseProfiles>
 
-		<flimlib.version>2.1.1</flimlib.version>
-		<flimj-ops.version>2.1.1</flimj-ops.version>
+		<flimlib.version>2.1.2-SNAPSHOT</flimlib.version>
+		<flimj-ops.version>2.1.2-SNAPSHOT</flimj-ops.version>
```
Comment (paddyroddy) [inline pom.xml:98]: These shouldn't change, right? We want users to pull from the cloud, not their local installation

File: pom.xml [hunk 2]
```diff
@@ -110,6 +111,8 @@
 			<groupId>flimlib</groupId>
 			<artifactId>flimlib</artifactId>
 			<version>${flimlib.version}</version>
+			<!-- <version>2.1.3-SNAPSHOT</version>
+			<classifier>natives-osx_arm64</classifier> -->
```
Comment (paddyroddy) [inline pom.xml:115]: I think we should just remove this altogether and hope there's an upstream fix at some point

File: src/main/java/flimlib/flimj/ui/controller/ConfigCtrl.java [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+            if (cfgSavePath != null) {
+                try {
+                    FileWriter writer = new FileWriter(cfgSavePath);
+                    String paramsJSONString = fp.getParams().toJSON();
+                    // add the binRadius to the config file
+                    String binRadiusJSONPrefix = jsonPrefix("binRadius");
+                    String paramsJSON = binRadiusJSONPrefix + fp.getBinRadius() + "," + paramsJSONString.substring(1);
+                    // add the fitType to the config file
+                    String fitTypeJSONPrefix = jsonPrefix("fitType");
+                    paramsJSON = fitTypeJSONPrefix + String.format("\"%s\"", fp.getAlgo()) + "," + paramsJSON.substring(1);
```
Comment (paddyroddy) [inline src/main/java/flimlib/flimj/ui/controller/ConfigCtrl.java:47]: Is `"\"%s\""` correct? I think so because `\` escapes the `"`?

**Review by paddyroddy — APPROVED** [review]
Looks good, just some questions about versioning

File: pom.xml (hunk 1 above)
Comment (drmatthews) [reply pom.xml]: Thanks for catching that - these should actually be `2.1.1`.

File: pom.xml (hunk 2 above)
Comment (drmatthews) [reply pom.xml]: I have now removed this as well.

File: src/main/java/flimlib/flimj/ui/controller/ConfigCtrl.java (hunk 3 above)
Comment (drmatthews) [reply src/main/java/flimlib/flimj/ui/controller/ConfigCtrl.java:47]: Yup, that's right.

**Pull request merged**